site/controllers/university.js

```diff
@@ -1,33 +1,25 @@
 const City = require('../models/city');
 
-exports.postAddUniversity = (req, res, next) => {
+exports.postAddUniversity = async (req, res, next) => {
   const cityName = req.body.city;
-  const universityName = req.body.universityName
+  const universityName = req.body.university
 
-  City.findOne({ where: { cityName: cityName  } })
-    .then((city) => {
-      console.log(city);
-      if (city) {
-        city.createUniversity({ universityName });
-      } else {
-        const city = City.build({cityName})
+  const [city, created] = await City.findOrCreate({
```

Comment:
I'm not a fan of creating a city instance in a university controller.
If the city doesn't exist, then the user should create it first and then create a university that belongs to it.

Reply:
from what I understand from our meetings the user will have a page for adding a university, so in the backend I will get the city name and university name when submitting the form( if this is not the expected functionality please explain it in our meeting today ).